topic-operator/src/test/java/io/strimzi/operator/topic/v2/TopicControllerIT.java

```diff
@@ -514,24 +533,6 @@ public void shouldCreateTopicInKafkaWhenManagedTopicCreatedInKube(KafkaTopic kt,
         createTopicAndAssertSuccess(kafkaCluster, kt);
     }
 
-    @Test
-    public void shouldCreateTopicInKafkaWhenKafkaTopicHasNoSpec(
```

Comment:
So we're changing a tested behavior. Are we _sure_ this isn't going to be a regression?

Reply:
Sure it is, but we are in alpha stage with very few users, so I guess this is fine. Also see my previous answer related to that.